### PR TITLE
Mention DLM in ignored docs test comments

### PR DESCRIPTION
**Pull request by n1v0lg**
Book-keeping PR to ensure that we don't prematurely un-ignore a doc tests suite.

### CODE REVIEW

File: x-pack/plugin/build.gradle [hunk 1]
```diff
@@ -95,7 +97,10 @@ tasks.named("yamlRestTestV7CompatTransform").configure { task ->
     "vectors/10_dense_vector_basic/Deprecated function signature",
     "to support it, it would require to almost revert back the #48725 and complicate the code"
   )
-  task.skipTest("vectors/20_dense_vector_special_cases/Indexing of Dense vectors should error when dims don't match defined in the mapping", "Error message has changed")
+  task.skipTest(
+    "vectors/20_dense_vector_special_cases/Indexing of Dense vectors should error when dims don't match defined in the mapping",
```
Comment (n1v0lg) [inline x-pack/plugin/build.gradle:101]: Auto-formatting

File: x-pack/docs/build.gradle [hunk 2]
```diff
@@ -1,4 +1,3 @@
-import org.elasticsearch.gradle.Version
```
Comment (n1v0lg) [inline x-pack/docs/build.gradle:1]: auto-formatting

**Review by andreidan — APPROVED** [review]
LGTM thanks Nikolaj

**Comment by n1v0lg** [comment]
@elasticmachine update branch

**Comment by n1v0lg** [comment]
@elasticmachine update branch

**Comment by n1v0lg** [comment]
@elasticmachine update branch bitte

**Pull request merged**